### PR TITLE
Improved fetching of notable violations

### PR DESCRIPTION
**Pull request by jbccollins**
- Fatalities and personal injuries are no longer considered 'interesting'
- Unfortunately no Hazmat incidents have location data :(

### CODE REVIEW

File: fetch.py [hunk 1]
```diff
@@ -29,8 +29,20 @@ def get_chat():
 
 
 def get_violations():
+    '''
+    This maps violation properties to a weight which is uses to determine
```
Comment (thachhoang) [inline fetch.py]: "used"

File: fetch.py [hunk 2]
```diff
@@ -29,8 +29,20 @@ def get_chat():
...
+        'alcohol': 1,
+        'belts': 1,
+        'property_damage': 1,
+        'work_zone': 2,
+        'hazmat': 2,
+        'arrest_type': 2,
+    }
     params = {
-        '$limit': 10,
+        '$limit': 50000,
```
Comment (thachhoang) [inline fetch.py:45]: You should commit the limit you used to generate the geojson file. Ignore if it's actually 50000.

Comment (jbccollins) [inline fetch.py:45]: It's actually 50000

File: fetch.py [hunk 3]
```diff
@@ -48,6 +75,7 @@ def get_violations():
         v['properties']['chat'] = get_chat()
         result.append(v)
 
+    logger.info(len(result))
```
Comment (thachhoang) [inline fetch.py]: More descriptive log: "X results".

**Comment by thachhoang** [comment]
No Hazmat incident has location data. :<

Do we know where they happen? Does it affect a large area so they don't bother giving it a point?

**Comment by jbccollins** [comment]
All the geolocation data for all hazmat violations are null. Idk why. We know the intersection so they happen at pretty specific places but parsing an intersection would be difficult.

**Pull request merged**